indico/web/client/js/react/components/notes/NoteEditor.jsx

```diff
@@ -14,35 +16,91 @@ import {handleSubmitError} from 'indico/react/forms';
 import {FinalModalForm} from 'indico/react/forms/final-form';
 import {Translate} from 'indico/react/i18n';
 import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+import {camelizeKeys} from 'indico/utils/case';
 
+import {FinalMarkdownEditor} from '../MarkdownEditor';
 import {FinalTextEditor} from '../TextEditor';
 
+// import './NoteEditor.module.scss';
+// import style manually
+import 'react-markdown-editor-lite/lib/index.css';
+
 export function NoteEditor({apiURL, imageUploadURL, closeModal, getNoteURL}) {
   const [currentInput, setCurrentInput] = useState(undefined);
   const [loading, setLoading] = useState(false);
   const [allowWithoutChange, setAllowWithoutChange] = useState(false);
+  const [renderMode, setRenderMode] = useState(undefined);
+
+  const combineNotes = (resp, mode) => {
+    if (mode === 'markdown') {
+      setCurrentInput(
+        resp.data.notes
+          .map(note => `#- ${_.escape(note.objectTitle)}\n${note.source}`)
+          .join('\n***\n\n')
```

Comment:
TIL that you can also use `***` to create a horizontal rule in markdown... I'd use the more commonly used `---` though